designs/2022-forbid-disable/README.md

```diff
@@ -0,0 +1,125 @@
+- Repo: eslint/eslint
+- Start Date: 2022-02-28
+- RFC PR:
+- Authors: Brian Bartels
+
+# ESLintRC Option To Forbid Disable For Specific Rules
+
+## Summary
+
+Allow users to forbid users from disabling certain rules. This is similar to `noInlineConfig` where the primary difference is that `noInlineConfig` is set for all rules.
+
+## Motivation
+
+When working with complex lint configs. Some rules are more important than other. For this reason it can be inconsequential to disable certain rules. But incredibly bad to disable others.
+
+EG:
+```
+//eslint-disable-next-line @government/required-SECURITY-rule
+```
+Is a much worse rule to disable than
+```
+//eslint-disable-next-line noExtraSemi
+```
+
+`noInlineConfig` works well for some. But in reality, many users disable rules at least somewhat frequently for valid reasons. And in large repos, there are going to be exceptions that need inline configuration. 
+
+## Detailed Design
+
+In `eslintrc` I will add another option for rules. The new state can be
```

Comment:
Just a reminder: eslintrc is frozen, and no more changes will be made to it. Your design should be built on the new config system. Please see lib/config/flat-config-array.js. The new config system is already available for testing in the Linter class by setting the condigType option to “flat”.

Reply:
Noted. The changes to the config should be minimal